### PR TITLE
This just cleans up the code, probably no fix

### PR DESCRIPTION
**Pull request by baudehlo**
WIP

Fixes #

Changes proposed in this pull request:
- 
- 

Checklist:
- [ ] docs updated
- [ ] tests updated
- [ ] [Changes](https://github.com/haraka/Haraka/blob/master/Changes.md) updated

### CODE REVIEW

**Comment by KingNoosh** [comment]
@baudehlo can you update your branch, please?

**Comment by msimerson** [comment]
I just used GitHub's resolve tool to (hopefully) fix the merge conflicts and not mangle the PR.

File: outbound/index.js [hunk 1]
```diff
@@ -320,12 +320,12 @@ exports.build_todo = function (todo, ws, write_more) {
...
+    const todo_json = JSON.stringify(todo, exclude_from_json);
+    const buf = new Buffer(4 + todo_json.length);
     buf.writeUInt32BE(todo_json.length, 0);
     buf.write(todo_json, 4);
 
-    var continue_writing = ws.write(buf);
+    const continue_writing = ws.write(buf);
     if (continue_writing) return process.nextTick(write_more);
 
     ws.once('drain', write);
```
Comment (msimerson) [inline outbound/index.js]: It doesn't appear write is defined. @baudehlo , should it have been renamed from write_more?

**Comment by baudehlo** [comment]
Oops yeah it should still be `write_more`.

**Pull request merged**